integrations/git_helper_for_declarative_test.go

```diff
@@ -37,7 +39,12 @@ func withKeyFile(t *testing.T, keyname string, callback func(string)) {
 	err = ssh.GenKeyPair(keyFile)
 	assert.NoError(t, err)
 
+	err = ioutil.WriteFile(path.Join(tmpDir, "ssh"), []byte("#!/bin/bash\n"+
```

Comment:
This will only run on Unix-like system? It didn't support windows.

Reply:
This is only for running the integration tests on git version <2.3.0

On all other git versions GIT_SSH_COMMAND will override GIT_SSH - you aren't able to run the integration tests on git 2.2.0 currently anyway on any system.

It's worth remembering that git should also run in an bash like environment, so it could actually work even on doze.

However, have you been unable to run the integration tests on windows with this change? It shouldn't have caused that - I guess I could stick a git version check around this though to be more explicit about what the hell it's actually doing.